.idea/runConfigurations/Build_Fishstick.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Build Fishstick" type="CargoCommandRunConfiguration" factoryName="Cargo Command">
```

Comment:
`Build Fishstick` isn't going to work perfectly with "build" run configurations for other chips. My thought was to have:
* `build/`
    * `build-fishstick.xml`
        * builds the `fishstick` crate that can also be run on host computer
        * `name="fishstick"` inside the file
    * `build-fishstick-stm32f401.xml`
        * builds the `fishstick-stm32f401` crate for hardware deployment
        * `name="fishstick-stm32f401"` inside the file
    * `build-fishstick-stm32f103.xml`
        * builds the `fishstick-stm32f103` crate for hardware deployment
        * `name="fishstick-stm32f103"` inside the file
* `deploy/`
    * `deploy-fishstick-stm32f401.xml`
        * deploys `fishstick-stm32f401` crate + `main.rs` bin on hardware
        * `name="fishstick-stm32f401"` inside the file
    * `deploy-fishstick-stm32f103.xml`
        * deploys `fishstick-stm32f103` crate + `main.rs` bin on hardware
        * `name="fishstick-stm32f103"` inside the file
        
The directory structure here is meant to imitate the run configuration folders you can add in the `Run Configuration` dialog. The disadvantage is that even if they visually appear in folders, the files don't get a directory structure. This is why I prefex `build-` and `deploy-` onto the filenames (this has to be done manually, but jetbrains does not get confused post-restart)

In the future we would also have a `tests/` folder

Reply:
would you be able to make this structure after merging? it would require testing for the stm32f103 and I dont have that. also it seems to be outside the scope of this PR

Reply:
Since we're _adding_ these run configurations, I figure the suggestion is in the scope of this PR. Lemme go and make the change for you on the branch